app/models/manageiq/providers/ansible_tower/configuration_manager.rb

```diff
@@ -16,6 +16,9 @@ class ManageIQ::Providers::AnsibleTower::ConfigurationManager < ManageIQ::Provid
            :with_provider_connection,
            :to => :provider
 
+  # FIXME: shouldnt be needed. Its here because otherwise the STI type column is the base class in refresh
+  has_many :configured_systems, :dependent => :destroy, :foreign_key => "manager_id"
```

Comment:
@Ladas can I somehow force the class on the collection right now? Or maybe we can tweak [get_entity_builder](https://github.com/durandom/manageiq/blob/11f9d5f79e8f7fd1521b2178f90c924173483a15/app/models/manager_refresh/save_collection/helper.rb#L101-L101)